website/src/css/newsletter.module.css

```diff
@@ -1,43 +1,103 @@
-.newsletterList {
-  list-style: none;
-  padding: 0;
-  margin: 0;
+.newsletterImage {
+  width: 95%;
+  max-width: 538px;
 }
 
-.newsletterList li {
-  padding-left: 1rem;
-  text-indent: -0.75rem;
+.bgGreen {
+  background-color: var(--ifm-color-primary-lightest);
```

Comment:
```suggestion
  background-color: rgba(35, 199, 181, 0.3);
```
Les icônes et le texte sont très sympa et donnent envie de lire :+1:  mais je trouve le background un peu trop 'présent', je te propose de lui rajouter une transparence (j'ai gardé la couleur 'ifm-color-primary-lightest' à laquelle j'ai appliqué une transparence), voici ce que ça donne :

![newsletter-bloc](https://user-images.githubusercontent.com/38593370/116271058-ef281a00-a77f-11eb-9cc4-73986fbe8343.png)
![newsletter-bloc-dark](https://user-images.githubusercontent.com/38593370/116271062-efc0b080-a77f-11eb-8ff0-8166696b0b78.png)